web-console/4.0-designs/labels/labels.md

```diff
@@ -0,0 +1,52 @@
+# Labels
+
+## Styles
+![pic 0](img/styles.png)
+
+- Label text should match standard link text color, with the light grey background to differentiate from other links
+- Label text color will not be matched to the corresponding resource badge color. All label colors will be the same across resource objects
```

Comment:
All the mockups (except for the 1st one) color the labels match with the resource badge color. So I assume this decision is made recently. Can anyone share the rationale behind this? Thanks

Reply:
@tlwu2013 I can update all screens with the new styles. We got input from the visual designers regarding text color to be more consistent and inline with other products. This should reduce the contrast/ accessibility issues we previously had with the badge color palette as well.